app/assets/stylesheets/modules/landing.scss

```diff
@@ -0,0 +1,21 @@
+.welcome {
+  background-image: image-url("plane_wing.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  min-height: 600px;
+
+  h2 {
+    color: orange;
+    text-align: center;
+    padding: 15px;
+  }
+
+  img {
+    width:auto;
```

Comment:
Properties should be ordered height, width<br>Colon after property should be followed by one space